### ISSUE TITLE
alt text for images in cards can be improved

### ISSUE THREAD

**Issue by Roshanjossey** [opensourcediversity/opensourcediversity.org#176]
Now, most images in cards have alt text of format "<title> image link".
eg: https://github.com/opensourcediversity/opensourcediversity.org/blob/main/index.html#L146

This can be improved to help people who rely on screen readers

Some guidelines for better alt text https://help.siteimprove.com/support/solutions/articles/80000863904-accessibility-image-alt-text-best-practices

**Comment by jancborchardt** [comment]
Hi @Roshanjossey, actually the project logos are decorative, and thus the alt text should be empty like: `alt=""`.
Otherwise we would duplicate text that is right below, which we already do with the project name currently. If you check with a screenreader, it will actually read the project name twice which is not intended.

See https://www.w3.org/WAI/tutorials/images/decorative/#example-2-decorative-image-as-part-of-a-text-link

**Comment by Roshanjossey** [comment]
Makes sense @jancborchardt

**Comment by athulmenondev** [comment]
I am a beginner and I want to contribute to opensource. Can I do this work?? Or is this work already taken?

**Comment by Ayfos** [comment]
Hola,

He visto que este issue/PR menciona añadir una etiqueta `<title>`. ¿Sigue abierto este trabajo o ya lo ha tomado alguien? Me gustaría contribuir si es posible.

¡Gracias!

**Comment by FiraBro** [comment]
Hi! I’d like to work on this issue. Could you please assign it to me?

**Comment by shirshamay** [comment]
hello, can I please help if this mission is not already assigned? Thank you.

**Comment by aman00077777** [comment]
Hello, i would love to work with this issue. if no one is assigned could i have the chance? I am a Mern stack web developer.

**Comment by Sarvan-Singh7** [comment]
I CAN ALSO CONTRIBUTE SEE MY PROFILE

**Comment by jancborchardt** [comment]
Anyone is free to pick this up and assignment is not needed. However because I just closed 2 pull requests opened back in September which were attempting this, please make sure to follow the comment above:
https://github.com/opensourcediversity/opensourcediversity.org/issues/176#issuecomment-3249408158

Thanks! :)

**Comment by Sarvan-Singh7** [comment]
please check it

**Comment by CodeByAfroj** [comment]
is this issue still open if open please assign it to me i would like to work on it

**Comment by jancborchardt** [comment]
@Sarvan-Singh7 I reviewed your pull request at https://github.com/opensourcediversity/opensourcediversity.org/pull/181 – please check. :) 

To everyone wanting to contribute: Please don’t use AI for this change, it is much easier and much less work to review to just go through the alt tags and empty them.

@CodeByAfroj please see my above comment: https://github.com/opensourcediversity/opensourcediversity.org/issues/176#issuecomment-3641695385

**Comment by Sarvan-Singh7** [comment]
Hi @jancborchardt,
Thank you for the clarification and for reviewing my pull request. I understand the point about decorative images and keeping the alt="" to avoid duplicate screen-reader output.

I’ll go through the code carefully and update the remaining alt tags accordingly, following your comment above. Please let me know if there’s anything specific you’d like me to double-check.

Thanks again for your guidance!

**Comment by Sarvan-Singh7** [comment]
Hi @jancborchardt,

I’ve updated the PR to ensure that all decorative project card images use `alt=""`, while keeping meaningful alt text for branding and functional icons (like the navbar logo and social links), as discussed in #176.

Could you please take another look when you have time?
Thanks for your guidance!

**Comment by Sarvan-Singh7** [comment]
Please Merge it Or tell me what i have to improve in my work ...
